package.json

```diff
@@ -1,37 +1,43 @@
 {
-    "name": "balance-text",
-    "version": "2.0.0",
-    "filename": "jquery.balancetext.min.js",
-    "homepage": "https://github.com/adobe-webplatform/balance-text",
-    "description": "A jQuery plugin for implementing balancing of text across lines in a web page",
-    "licenses": [
-        {
-            "type": "Apache License, Version 2.0",
-            "url": "http://www.apache.org/licenses/LICENSE-2.0"
-        }
-    ],
-    "maintainers": [
-        {
-            "name": "Adobe Systems, Inc.",
-            "web": "https://github.com/adobe-webplatform"
-        }
-    ],
-    "repositories": [
-        {
-            "type": "git",
-            "url": "https://github.com/adobe-webplatform/balance-text"
-        }
-    ],
-    "dependencies": {
-        "jquery": "1.9.1 - 3"
-    },
-    "autoupdate": {
-        "source": "git",
-        "target": "git://github.com/adobe-webplatform/balance-text.git",
-        "basePath": "",
-        "files": [
-            "jquery.balancetext.js",
-            "jquery.balancetext.min.js"
-        ]
+  "name": "balance-text",
+  "version": "3.0.0",
+  "filename": "jquery.balancetext.min.js",
```

Comment:
I have this lib hosted on a couple CDNs which auto-update when a new github Release is created. The `filename` field needs to be updated for https://cdnjs.cloudflare.com and `main` needs to be updated for https://cdn.jsdelivr.net to `balancetext.min.js`.

The thing I'm not quite sure about is whether exsiting libs for version <= 2.0.0 will be maintained. Does anyone know how the CDN auto-updating  work?